Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/DemoTargetPositioning/Scripts/MoveObjByEyeGaze.cs

```diff
@@ -30,10 +30,7 @@ private IMixedRealityInputSystem InputSystem
         {
             get
             {
-                if (inputSystem == null)
-                {
-                    MixedRealityServiceRegistry.TryGetService<IMixedRealityInputSystem>(out inputSystem);
-                }
+                MixedRealityServiceRegistry.TryGetService<IMixedRealityInputSystem>(out inputSystem);
```

Comment:
nit: use core services

Reply:
@sostel

Reply:
Changed to use CoreServices.Input. Thanks!